src/app/content/reducer.ts

```diff
@@ -18,6 +18,8 @@ import searchReducer, {initialState as initialSearchState } from './search/reduc
 import studyGuidesReducer, {initialState as initialStudyGuidesState } from './studyGuides/reducer';
 import { State } from './types';
 
+const storedTextSize = typeof window !== 'undefined' && window.localStorage.getItem('textSize');
```

Comment:
you may need to have this here if my other suggestion causes a flash of the stuff at a different size, but even then the key should at least be stored in a constant